corehq/apps/export/utils.py

```diff
@@ -51,3 +52,22 @@ def get_export(export_type, domain, export_id=None, username=None):
             SMSExportDataSchema.get_latest_export_schema(domain, include_metadata)
         )
     raise Exception("Unexpected export type received %s" % export_type)
+
+
+def get_or_create_default_export_settings_for_domain(domain):
```

Comment:
I do think this is a fine django app for this function to live in. Export seems more logical than accounting. The other place I could see it living is the new enterprise app, but export is also fine.